File: harbor-webhook/database/images.go
package database

import "encoding/json"

type Images struct {
	ID           uint        `gorm:"primarykey"`
	RepoFullName string      `gorm:"size:50"`
	Tag          string      `gorm:"size:30"`
	Namespace    string      `gorm:"size:30"`
	Name         string      `gorm:"size:30"`
	ResourceUrl  string      `gorm:"size:100;uniqueIndex:images_unique"`
	Digest       string      `gorm:"size:255"`
	RepoType     string      `gorm:"type:char(255)"`
	DateCreate   json.Number `gorm:"type:char(20)"`
}

File: algorithm-scheduling-server/src/main/java/cn/hanglok/algoSched/entity/Images.java
package cn.hanglok.algoSched.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import java.io.Serializable;
import lombok.Getter;
import lombok.Setter;

/**
 * <p>
 * 算法镜像表
 * </p>
 *
 * @author Allen
 * @since 2024-03-01
 */
@Getter
@Setter
public class Images implements Serializable {

    private static final long serialVersionUID = 1L;

    /**
     * 自增 id
     */
    @TableId(value = "id", type = IdType.AUTO)
    private Long id;

    private String namespace;

    private String name;

    private String repoFullName;

    private String tag;

    private String resourceUrl;

    private String digest;

    private String repoType;

    private String dateCreate;

    private String label;
}
